scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+set +o pipefail
+
+python -m pip install --upgrade pip
+echo docker-hub==2.2.0 > dev-requirements.txt
+pip install -r dev-requirements.txt
```

Comment:
`install_ci_dependencies()`?

Reply:
done

Reply:
@oriyarde I suggest resolving finished conversations (there are more below this one)

Reply:
@zingero I haven't approved the PR yet and the conversations shouldn't be resolved before I finish validating them

Reply:
I didn't ask you to do that immediately.
The conversations are two weeks old with a "done" status.

Reply:
two weeks is indeed long, but this PR is not prioritized, while other PRs are very active, so IMO it's a trade-off worth having

Reply:
in addition, there is a lot of common code between the PRs, so it's easier to first finish with others, and then wait for this one to sync before continuing the review